lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx

```diff
@@ -479,6 +499,11 @@ export default function LaunchErrorDialog({
               account to one of this assignment&apos;s groupings.
             </b>
           </p>
+          <p>
+            <ExternalLink href="https://web.hypothes.is/help/troubleshooting-lms-app-error-messages-in-moodle/">
+              Troubleshooting LMS App Error Messages in Moodle
+            </ExternalLink>
```

Comment:
Perhaps it's worth extracting these to a reusable component, as it's always the same.

It can be a local component to this file, for simplicity.

```tsx
function MoodleKBLink() {
  return(
    <ExternalLink href="https://web.hypothes.is/help/troubleshooting-lms-app-error-messages-in-moodle/">
      Troubleshooting LMS App Error Messages in Moodle
    </ExternalLink>
  );
}
```

```suggestion
          <p>
            <MoodleKBLink />
```